addons/mrp/models/mrp_routing.py

```diff
@@ -77,3 +77,9 @@ def _compute_workorder_count(self):
         count_data = dict((item['operation_id'][0], item['operation_id_count']) for item in data)
         for operation in self:
             operation.workorder_count = count_data.get(operation.id, 0)
+
+    def _get_comparison_values(self):
+        if not self:
+            return False
```

Comment:
test already done in `ensure_one`

Reply:
`ensure_one` raises an error when the record is empty, so I'm doing this in order to avoid checking if it's not empty in every function call

Reply:
`ensure_one` raises an error when the record is empty, so I'm doing this in order to avoid checking if it's not empty in every function call